server_addon/houdini/client/ayon_houdini/plugins/publish/collect_render_products.py

```diff
@@ -131,12 +132,16 @@ def replace(match):
             self.log.debug("Render Product %s%s", aov_label, prim_path)
             self.log.debug("Product name: %s", filename)
 
+
+        # As long as we have one AOV then multipartExr should be True.
+        instance.data["multipartExr"] = len(files_by_product) <= 1
```

Comment:
This is technically not entirely true. It may just not have multiple AOVs and is then not multipart, right? USD Render Products are also completely capable of writing no beauty but only e.g. a Z-pass (even though you'd usually not do that I suppose)
Out of curiosity, why do we need this?

Reply:
> why do we need this?

Well, searching for `"multiPartExr"` in this file [pyblish_functions.py](https://github.com/ynput/ayon-core/blob/develop/client/ayon_core/pipeline/farm/pyblish_functions.py) can answer the question. 

As far as I was able to understand the logic. if there's one AOV, the logic will assume it's reviewable.
And I was merely trying to adopt the same logic.